openspec/changes/fix-error-message-clarity/design.md

```
@@ -0,0 +1,39 @@
+## Context
+
+`display_error()` formats the `anyhow` error chain as:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)